### DISCUSSION TITLE
BackButton is not supported in version 6.0

### DISCUSSION THREAD

**Discussion by milushov** [vkruglikov/react-telegram-web-app#33] [Q&A] [answered]
I don't know if it's related or not.

I see this warning in the console:

```
telegram-web-app.js:622 [Telegram.WebApp] BackButton is not supported in version 6.0
```

How is it possible that I'm using an outdated version of the script when I load it from the URL https://telegram.org/js/telegram-web-app.js?

**Comment by vkruglikov** [comment, marked as answer]
Hi @milushov 

It may depend on the version telegram client. 
You use the last version telegram-web-app.js SDK, but this is not enough.

If you use the desktop Telegram client, I recommend you update https://desktop.telegram.org/ to the last version.

Keep in mind that your users may also experience this problem if they have an old version of the telegram

**Comment by tiditik** [comment]
I updated telegram to the latest version and it still gives me the error.
Indeed, my current version is 6.0 , so says WebApp, but how do I upgrade to something that is not there?
```
6.0
telegram-web-app.js:1476  [Telegram.WebApp] Method showPopup is not supported in version 6.0

```

**Reply by OybekDeveloper** [reply]
I have the same problem, please help

**Comment by OybekDeveloper** [comment]
Bro, if you have found a solution to this problem, please help me

my telegram account:@coderbux

pls help meee!!!!!

**Comment by Umar-Juraev** [comment]
pnite menya toje

**Comment by GizziXZ** [comment]
I'm also having this same problem
